src/shared/components/modals/InputModal.tsx

```diff
@@ -38,20 +40,32 @@ const InputModal: FunctionComponent<InputModalProps> = ({
 }) => {
 	const [input, setInput] = useState<string>(inputValue || '');
 
-	const handleConfirm = () => {
+	// Listeners
+	const onClickClose = () => {
+		onClose();
+		setInput(inputValue || '');
+	};
+
+	const onClickConfirm = () => {
+		if (!input) {
+			toastService('Gelieve een collectie-titel in te vullen.', TOAST_TYPE.DANGER);
```

Comment:
Die toast message mss als een prop zetten anders gaat deze component niet echt re-usable zijn?
Of als deze collection specififek is mss deze in die module zetten ipv in shared